src/modules/git/index.ts

```diff
@@ -102,6 +104,8 @@ export class Git {
    * faker.git.shortSha() // '6155732'
    */
   shortSha(): string {
-    return this.faker.datatype.hexadecimal({ length: 7, case: 'lower' });
+    return this.faker.datatype
+      .hexadecimal({ length: 7, case: 'lower' })
+      .substring(2);
```

Comment:
Use `prefix` instead of string manipulation.

```suggestion
      .hexadecimal({ length: 7, case: 'lower', prefix: '' });
```